internal/corazawaf/transaction.go

```diff
@@ -650,15 +647,15 @@ func (tx *Transaction) ExtractArguments(orig types.ArgumentType, uri string) {
 	data := urlutil.ParseQuery(uri, '&')
 	for k, vs := range data {
 		for _, v := range vs {
-			tx.AddArgument(orig, k, v)
+			tx.AddRequestArgument(orig, k, v)
 		}
 	}
 }
 
-// AddArgument Add arguments GET or POST
+// AddRequestArgument Add request arguments GET or POST
 // This will set ARGS_(GET|POST), ARGS, ARGS_NAMES, ARGS_COMBINED_SIZE and
 // ARGS_(GET|POST)_NAMES
-func (tx *Transaction) AddArgument(argType types.ArgumentType, key string, value string) {
+func (tx *Transaction) AddRequestArgument(argType types.ArgumentType, key string, value string) {
```

Comment:
What do you think of `AddGetRequestArgument`, `AddPostRequestArgument`, `AddPathRequestArgument`? I don't know if having this enum is very useful and requires looking up a second API to be able to use the functionality